hack/update-deployments-with-latest-image.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+if ! command -V skopeo; then
+    echo "skopeo required!"
+    exit 1
+fi
+
+if [ $# -gt 1 ]; then
+    if [ ! -d "$2" ]; then
+        echo "$2 is not a directory!"
+        exit 1
+    fi
+    job_dir="$2"
+else
+    job_dir="$(readlink --canonicalize "$(cd "$(cd "$(dirname "$0")" && pwd)"'/../github/ci/prow-deploy/kustom/base/manifests/local' && pwd)")"
```

Comment:
nit: I guess job_dir is not the best name here if it is now aimed at deployments?

Reply:
@brianmcarey done, PTAL, thanks for your review!